packages/react-jsx-highcharts/src/components/Chart/Chart.js

```diff
@@ -5,31 +5,30 @@ import useModifiedProps from '../UseModifiedProps';
 import useChart from '../UseChart';
 import useManualEventHandlers from '../UseManualEventHandlers';
 
-const Chart = memo(({ type = 'line', ...restProps }) => {
+const Chart = memo(({ type = 'line', width, height, ...restProps }) => {
   const chart = useChart();
   const mounted = useRef(false);
 
   const modifiedProps = useModifiedProps({ type, ...restProps });
 
+  useEffect(() => {
+    if (!(typeof width === 'undefined' && typeof height === 'undefined')) {
```

Comment:
Just curious why `typeof x === 'undefined'` instead of `x === undefined`?

Reply:
I wasn't thinking. It just came that way :) 

Should I change it?

Reply:
I changed it to ```x === undefined```